march_hardware/src/MarchRobot.cpp

```diff
@@ -47,6 +47,16 @@ void MarchRobot::startEtherCAT()
     return;
   }
   ethercatMaster.start(this->jointList);
+
+  if (do_reset_imc)
+  {
+    ROS_INFO("Resetting all IMC");
```

Comment:
```suggestion
    ROS_INFO("Resetting all IMCs");
```

Reply:
You have this comment twice now: here and in resetIMC(). I would remove it here.